Package.swift

```diff
@@ -1,9 +1,14 @@
-// swift-tools-version: 5.6
+// swift-tools-version: 5.10
 // The swift-tools-version declares the minimum version of Swift required to build this package.
 
 import PackageDescription
 import Foundation
 
+let swiftSettings: [SwiftSetting] = [
+    .enableExperimentalFeature("StrictConcurrency"),
+    .unsafeFlags(["-strict-concurrency=complete"]),
```

Comment:
I'm not sure, but is the unsafeFlags necessary?

Reply:
Yes, I also thought it's fine only with `ExperimentalFeature` but it seems to need the `unsafeFlags` too.